README.md

```diff
@@ -353,6 +351,38 @@ The default histogram implementation uses a random sampling algorithm with expon
 This was authored by Johan Buratti. 
 
 
+## Distribution
+An aggregatable data structure, T-digest, that allows for aggregating percentiles. To get accurate percentiles, you need to setup good bucket boundaries.
```

Comment:
So the advantage of T-digest is that you don't need to provide any binning, bucket configuration is not required.
check [this description ]( https://github.com/spotify/semantic-metrics/blob/master/core/src/main/java/com/spotify/metrics/core/Distribution.java#L31)

Reply:
Thanks for sharing that @ao2017 I used it to re-write the first part. Should be good for another review now